wazo_confd/plugins/user_group/resource.py

```diff
@@ -2,43 +2,52 @@
 # SPDX-License-Identifier: GPL-3.0-or-later
 
 from flask import request
-from marshmallow import fields
-
-from xivo_dao.helpers import errors
-from xivo_dao.helpers.exception import NotFoundError
+from marshmallow import fields, validates_schema, post_load
+from marshmallow.exceptions import ValidationError
 
 from wazo_confd.auth import required_acl
 from wazo_confd.helpers.mallow import BaseSchema, Nested
 from wazo_confd.helpers.restful import ConfdResource
 
 
-class GroupSchemaIDLoad(BaseSchema):
-    id = fields.Integer(required=True)
+class GroupSchemaIDUUIDLoad(BaseSchema):
+    id = fields.Integer()
+    uuid = fields.UUID()
+
+    @validates_schema
+    def _validate_at_least_one_identifier(self, data, **kwargs):
+        _id = data.get('id')
+        uuid = data.get('uuid')
+
+        provided_identifiers = [identifier for identifier in [_id, uuid] if identifier]
+        if len(provided_identifiers) == 0:
+            raise ValidationError('One identifier(id or uuid) must be provided')
 
+    @post_load
+    def remove_identifier_not_provided(self, data, **kwargs):
+        for k in ['id', 'uuid']:
+            try:
+                if not data[k]:
+                    del data[k]
+            except KeyError:
+                pass
+        return data
```

Comment:
Please use marshmallow built-in [partial](https://marshmallow.readthedocs.io/en/stable/marshmallow.schema.html#schema) feature to make this logic